sdks/go/pkg/beam/io/bigqueryio/bigquery.go

```diff
@@ -28,12 +28,15 @@ import (
 	"cloud.google.com/go/bigquery"
 	"github.com/apache/beam/sdks/go/pkg/beam"
 	"github.com/apache/beam/sdks/go/pkg/beam/core/util/reflectx"
+	bq "google.golang.org/api/bigquery/v2"
```

Comment:
Is it safe to combine the cloud.google.com API with the V2 API?

Be warned: It's not currently simple to add new Go dependencies, due to gradle and jenkins etc, the lock file will need updating if the v2 package isn't already getting vendored.
There's a low priority task to move to Go Modules instead which would make it easier for everyone involved, but that's a hairy mess WRT beam right now. I can provide more details if necessary.

Reply:
Internally the cloud api includes v2 api, so I would assume it would be safe, but honestly, this is beyond my depth of the overall build system.

Reply:
I've almost got the right grasp on it to fix it, but we can give it a go.
If Jenkins starts failing, we can revert the PR and re-evaluate.